frontend/metrics_program.html

```diff
@@ -20,9 +16,7 @@
 <option value="ts_users" data-sizex="8" data-sizey="2">Timeseries of active users</option>
 <option value="ts_all_modes" data-sizex="8" data-sizey="2">Timeseries of all mode shares</option>
 <!-- Mode Specific -->
-<option value="ntrips_${data.intro.mode_studied}_purpose" data-sizex="4" data-sizey="4">`${data.intro.mode_studied}` specific trip count by purpose</option>
-<option value="miles_${data.intro.mode_studied}_replaced_mode" data-sizex="4" data-sizey="4">`${data.intro.mode_studied}` specific trip length (${data.display_config.use_imperial}) by replaced mode</option>
-<option value="average_miles_replaced_mode" data-sizex="4" data-sizey="4">Average Miles for each replaced mode (w/Other)</option>
+<option value="average_miles_replaced_mode" data-sizex="6" data-sizey="4">Average Miles for each replaced mode (w/Other)</option>
```

Comment:
Yes, I have updated the values to use dynamically select between km and miles as applicable.
I also thought about changing the option value to use more generic term like `distance` instead of `miles`. But I didn't wanted to push changes which aren't really part of this PR. What do you think about it?

Reply:
A lot of the what I had was "Average trip length (unit) ..." so I think it would be ok to change it to `Average trip length (unit) for each replaced mode` or something similar, or leave it, whatever is most consistent with the rest of the chart names.

Thank you for making the update, I'm not sure how that one slipped past me.